Documentation/guides/building-apps/build-items.md

```diff
@@ -222,8 +222,36 @@ This is so MSBuild does not try to interpret them as actual file wildcards.
 NOTE: `*`, `?` and `.` will be replaced in the `BuildApk` task with the
 appropriate RegEx expressions.
 
+If the default file glob is too restrictive you can remove it by adding the 
+following to your csproj
+
+```
+<ItemGroup>
+	<AndroidPackagingOptionsExclude Remove="$([MSBuild]::Escape('*.kotlin_*')" />
+</ItemGroup>
+```
+
 Added in Xamarin.Android 13.1 and .NET 7.
 
+## AndroidPackagingOptionsInclude
+
+A set of file glob compatible items which will allow for items to be
+included from the final package. The default values are as follows
+
+```
+<ItemGroup>
+	<AndroidPackagingOptionsInclude Include="$([MSBuild]::Escape('*reflect.kotlin_builtins')" />
+</ItemGroup>
+```
+Items can use file blob characters for wildcards such as `*` and `?`.
+However these Items MUST use URL encoding or '$([MSBuild]::Escape(''))'.
+This is so MSBuild does not try to interpret them as actual file wildcards.
+
+NOTE: `*`, `?` and `.` will be replaced in the `BuildApk` task with the
+appropriate RegEx expressions.
```

Comment:
It's probably best to *not* mention regular expressions, and just stick with file globs.